src/Compilers/CSharp/Portable/Symbols/SymbolWithAnnotations.cs

```diff
@@ -182,6 +185,19 @@ public TypeSymbolWithAnnotations AsNullableReferenceOrValueType(CSharpCompilatio
         /// simply returns a symbol for the reference type.
         /// </summary>
         public abstract bool? IsNullable { get; }
+        public abstract bool IsAnnotated { get; }
+
+        // AnnotationKind considers IsAnnotated and NonNullTypes only. Compare with
```

Comment:
Consider moving or duplicating this comment up to `IsAnnotated` and make it an xml doc.